utils/build-presets.ini

```diff
@@ -1453,6 +1453,8 @@ skip-build-tvos
 skip-test-tvos
 skip-build-watchos
 skip-test-watchos
+skip-build-xros
```

Comment:
Intentional?

Reply:
Yes! Piggybacking with the optimizer change.